### PR TITLE
Video series

### PR DESCRIPTION
**Pull request by jmelvnsn**

### CODE REVIEW

**Comment by spra85** [comment]
@jmelvnsn, we should pull in latest changes from master to make sure nothing broke stuff.

**Comment by spra85** [comment]
There's a js file flagged as being renamed from `__init__.py` ? -> bulbs/videos/migrations/__init__.py → ...deo-recirc-list/video-recirc-list.test.js

File: bulbs/videos/static/videos/js/series/main.browserify.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+var SeriesMeta = require('./series-meta/series-meta');
+var SeriesVideoGrid = require('./series-video-grid/series-video-grid');
+var LatestEpisode = require('./series-video-latest-episode/series-video-latest-episode');
+var SeriesVideoList = require('./series-video-list/series-video-list');
+
+SeriesPage = function() {
```
Comment (spra85) [inline bulbs/videos/static/videos/js/series/main.browserify.js]: This should be a separate file included via `require`, not inlined in the `main.browserify.js`

File: bulbs/videos/static/videos/js/series/series-meta/series-meta.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+SeriesMeta.prototype.fetchSeriesMeta = function() {
+  $.getJSON(this.source, this.seriesMetaFetched.bind(this));
+};
+
+SeriesMeta.prototype.seriesMetaFetched = function(data) {
+  $('#series-description').html(data.series_description);
+  this.$seriesTitle.html(data.series_name);
+  /* Commenting this out until we resolve logo crop issue
```
Comment (spra85) [inline bulbs/videos/static/videos/js/series/series-meta/series-meta.js]: Still something that needs to be looked at?

File: bulbs/videos/templates/videos/series-page.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<header>
+  <h1 id="series-title"></h1>
+  <p>Watch The Latest Episode</p>
+</header>
+
+<figure class="video-container">
+  <div class="content">
+    <bulbs-video
+      twitter-handle="avclub"
+      src="{{ VIDEOHUB_BASE_URL }}/video/"
```
Comment (spra85) [inline bulbs/videos/templates/videos/series-page.html]: I feel like the `latest-episode` js module should actually add the `bulbs-video` element to the page.  Wouldn't this try to first make a request to the base url before latest episode appends the `<video-id>.json` ?  Maybe @collin can weigh in too

File: bulbs/videos/templates/videos/video-series-detail-test.html [hunk 4]
```diff
@@ -2,4 +2,4 @@
 {{ series_name }}
 {{ series_description }}
 {{ total_seasons }}
-{{ total_episdoes }}
+{{ total_episdoes }}
```
Comment (spra85) [inline bulbs/videos/templates/videos/video-series-detail-test.html]: Spelling.

File: test_helper.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    return spy;
+  },
+
+  stub: function(object, method, retVal) {
+     var stub = sinon.stub(object, method).returns(retVal);
+     stubs.push(stub);
+     return stub;
+  },
+
+  click: function(el){
```
Comment (spra85) [inline test_helper.js]: We should be able to actually remove this `click` function

File: bulbs/videos/static/videos/js/series/series-meta/series-meta.js (hunk 2 above)
Comment (jmelvnsn) [inline bulbs/videos/static/videos/js/series/series-meta/series-meta.js]: This requires that our design team create standard crops for all series logos and re-upload them to Onions Studios. Kelly is aware of the issue and we chose to launch without the logo

File: bulbs/videos/views.py [hunk 6]
```diff
@@ -9,7 +9,7 @@
 class SeriesDetailView(DetailView):
 
     def get_template_names(self):
-        template_names = ["videos/series-detail.html"]
+        template_names = ["videos/video-series-detail-test.html"]
```
Comment (mparent61) [inline bulbs/videos/views.py]: Should remove "-test" suffix

Comment (spra85) [inline bulbs/videos/views.py]: Good catch @mparent61 ... 👍

File: bulbs/videos/templates/videos/series-page.html (hunk 3 above)
Comment (collin) [inline bulbs/videos/templates/videos/series-page.html]: That sounds correct, this will make a request to `//www.onionstudios.com/video/`

**Pull request merged**